examples/open_ai_function_calls.rb

```diff
@@ -0,0 +1,41 @@
+require "langchain"
```

Comment:
@michelson Btw -- I ran this example and the other one and they both threw undefined errors method. Would you please double-check?

Reply:
it works on my end, and it works on CI too it seems

Reply:
The CI doesn't run the examples. They're meant to be run locally with "$ ruby examples/open_ai_function_calls.rb"

Reply:
ok, and yes, examples works, do you have the qdrant configurated to some index?

Reply:
I do `bundle exec ruby  ./examples/open_ai_function_calls.rb` and `bundle exec ruby  ./examples/open_ai_qdrant_function_calls.rb` both work